package.json

```diff
@@ -8,6 +8,24 @@
   "main": "dist/a11y-dialog.js",
   "module": "dist/a11y-dialog.esm.js",
   "types": "dist/a11y-dialog.d.ts",
+  "exports": {
+    ".": {
```

Comment:
As discussed elsewhere, I guess the `"."` character is optional; I’m just used to seeing this syntax more. Probably doesn’t matter.